Resources/Prototypes/Recipes/Construction/Graphs/lighting.yml

```diff
@@ -0,0 +1,46 @@
+﻿- type: constructionGraph
```

Comment:
@Zumorica do you mind me pinging you whenever construction comes up in a review?